### PR TITLE
Blog post: The real weakest link in software supply chain security

### PR DESCRIPTION
**Pull request by rebeccadee**
PREVIEW: https://deploy-preview-5205--sourcegraph.netlify.app/blog/real-weakest-link-in-software-supply-chain-security/

https://app.asana.com/0/1199959077348208/1200076936776936/f

### CODE REVIEW

File: blogposts/2022/third-party-vulnerabilties-process.md
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:10]: TODO

File: blogposts/2022/third-party-vulnerabilties-process.md
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:36]: TODO: Add link to case study

File: blogposts/2022/third-party-vulnerabilties-process.md
Comment (rebeccadee) [inline blogposts/2022/third-party-vulnerabilties-process.md:56]: TODO

**Comment by rebeccadee** [comment]
@nickmyyz I will pop in tomorrow to add the images and resolve outstanding todos, but would be super helpful to get an approving review from you in the meantime so I'm not blocked from merging tomorrow AM! 🙏 TY!

File: blogposts/2022/third-party-vulnerabilties-process.md
Comment (rebeccadee) [reply blogposts/2022/third-party-vulnerabilties-process.md:56]: @jennybergen I think the answer is no, but do you know if we have something to link to here yet?

**Review by nickmyyz — APPROVED** [review]
I read it over again and still think it's ready to go (minus the to-dos of course)! The only thing I might take a look at is whether you could add more paragraph breaks here and there (the intro paragraph for example is a bit long) but I wouldn't consider that blocking.

File: blogposts/2022/third-party-vulnerabilties-process.md
Comment (rebeccadee) [reply blogposts/2022/third-party-vulnerabilties-process.md:56]: I will come back and add this when we have it

**Comment by rebeccadee** [comment]
@nickmyyz / @andreeleuterio I didn't realise that making further changes to the PR would invalidate the previous approval 🙄  Would one of you be able to approve and merge today please? 🙏 I understand why we want to enforce reviews before merging but this is proving to be a bit of a pain!!

**Review by andreeleuterio — DISMISSED** [review]

**Comment by andreeleuterio** [comment]
@rebeccadee I got you

**Review by amieroth — APPROVED** [review]

**Pull request merged**
